tests/acceptance/features/bootstrap/CliContext.php

```diff
@@ -139,6 +139,35 @@ public function theAdministratorRemovesTheVersionsOfFileUsingFileId($file, $user
 		$this->featureContext->setResponse(CliHelper::runCommand($body));
 	}
 
+	/**
+	 * @When /^the administrator reindexes all spaces using the CLI$/
+	 *
+	 * @return void
+	 */
+	public function theAdministratorReindexesAllSpacesUsingTheCli(): void {
+		$command = "search index --all-spaces";
+		$body = [
+			"command" => $command
+		];
+		$this->featureContext->setResponse(CliHelper::runCommand($body));
+	}
+
+	/**
+	 * @When /^the administrator reindexes a space "([^"]*)" using the CLI$/
+	 *
+	 * @param string $spaceName
+	 *
+	 * @return void
+	 */
+	public function theAdministratorReindexesASpaceWithSpacenameUsingTheCli(string $spaceName): void {
+		$spaceId = $this->featureContext->spacesContext->getSpaceIdByName($this->featureContext->getAdminUsername(), $spaceName);
```

Comment:
`spacesContext` is available in this context
```suggestion
		$spaceId = $this->spacesContext->getSpaceIdByName($this->featureContext->getAdminUsername(), $spaceName);
```